extensions/gdx-freetype/src/com/badlogic/gdx/graphics/g2d/freetype/FreeTypeFontGenerator.java

```diff
@@ -164,8 +162,14 @@ public BitmapFont generateFont (FreeTypeFontParameter parameter) {
 	 * @param parameter configures how the font is generated */
 	public BitmapFont generateFont (FreeTypeFontParameter parameter, FreeTypeBitmapFontData data) {

```

Comment:
I know this method wasn't added in your PR, in fact I added it so FreeTypeBitmapFontData can be subclassed to override methdos, but this feels weird. Probably would be better for FreeTypeBitmapFontData to take a FreeTypeFontGenerator rather than have FreeTypeFontGenerator create or populate a FreeTypeBitmapFontData. Thoughts? If we agree maybe I do this after your PR is merged.